### DISCUSSION TITLE
Maximum time length of audiobook

### DISCUSSION THREAD

**Discussion by Pantyboy** [enzo1982/freac#382] [Help & Support] [answered]
I'm trying to create an audiobook and .cue file from 27 separate .mp3 files. The total length of the book is 14 hours 55 minutes.

The .cue file is fine until it gets to files 25, 26, and 27, then it starts showing errors. Below is a selection from the generated .cue file.

>...
...
TRACK 24 AUDIO
  TITLE "24"
  INDEX 01 808:20:58
TRACK 25 AUDIO
  TITLE "25"
  INDEX 01 851:97416:7304398
TRACK 26 AUDIO
  TITLE "26"
  INDEX 01 861:97423:7304440
TRACK 27 AUDIO
  TITLE "27"
  INDEX 01 873:97402:7304374

Any idea what's causing this? Is it because _there is_ a maximum length of time for an audiobook?

Many thanks in advance.
Martin

**Comment by enzo1982** [comment, marked as answer]
Thank you for bringing this up!

This was a bug in the cuesheet generator. Fixed now in the latest [continuous build](https://github.com/enzo1982/freac/releases/tag/continuous).

**Reply by Pantyboy** [reply]
Excellent. Many thanks.
